.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: c
 
 # Ubuntu 14.04 LTS
-dist: trusty
+dist: xenial
 
 sudo: required
 
 install:
 - sudo apt-get -qq update
-- sudo apt-get -y  install libelf-dev libstdc++-4.8-dev pandoc
+- sudo misc/install-deps.sh -y
 
-script: ./configure && make
+script: ./configure && make && tests/unittest
```

Comment:
It won't build the unittest binary so will fail.  I think you can call `make test TESTARG=unittest` instead.

Reply:
Oh, my mistake. I changed this to 'make uniitest'

Reply:
Oh, you're right